kaiax/staking/README.md

```diff
@@ -145,6 +148,7 @@ Query the StakingInfo to be used for the block `num`.
 - Returns
   - `StakingInfoResponse`
 - Example
+
 ```json
```

Comment:
Would you additionally update the JSON-RPC example to include clStakingInfos?

Reply:
Thanks. [5e340df](https://github.com/kaiachain/kaia/pull/265/commits/5e340dff723df7f2047b0176902583f66dd3eb1c)